app/concepts/api/v1/comments/operation/destroy.rb

```diff
@@ -0,0 +1,13 @@
+module Api::V1
+  module Comments::Operation
+    class Destroy < Trailblazer::Operation
+      step Model(Comment, :find)
+      step Policy::Guard(Lib::Policy::UserGuard.new), fail_fast: true
+      step :destroy
+
+      def destroy(ctx, **)
+        ctx['model'].destroy!
```

Comment:
It's not okay to raise an exception in operation. See explanation why `find_by` is used:
http://trailblazer.to/gems/operation/2.0/api.html#model-findby
If you use bang methods you should add a rescue: http://trailblazer.to/gems/operation/2.0/api.html#rescue